<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PHoleMUD</class>
 <widget class="QWidget" name="PHoleMUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="MPLCanvas2" name="w_viewer" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="b_dxf">
        <property name="text">
         <string>Define Slot from DXF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WPathSelectorV" name="w_path_json" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="g_mat">
        <property name="title">
         <string>Materials</string>
        </property>
        <layout class="QVBoxLayout" name="g_mat_layout">
         <item>
          <widget class="WMatSelect" name="w_mat_0" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>179</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="g_output">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="out_slot_surface">
           <property name="text">
            <string>Hole full surface : ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="out_magnet_surface">
           <property name="text">
            <string>Hole magnet surface : ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="out_Rmin">
           <property name="text">
            <string>Rmin : ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="out_Rmax">
           <property name="text">
            <string>Rmax : ?</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WPathSelectorV</class>
   <extends>QWidget</extends>
   <header>......GUI.Tools.WPathSelector.WPathSelectorV</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MPLCanvas2</class>
   <extends>QWidget</extends>
   <header>......GUI.Tools.MPLCanvas</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WMatSelect</class>
   <extends>QWidget</extends>
   <header>......GUI.Dialog.DMatLib.WMatSelect.WMatSelect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Resources/pyleecan.qrc"/>
  <include location="../../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
